components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -9186,7 +9266,7 @@ public void removeThrottlePolicy(String policyLevel, String policyName, int tena
                 globalPolicy.setDescription(rs.getString(ThrottlePolicyConstants.COLUMN_DESCRIPTION));
                 globalPolicy.setPolicyId(rs.getInt(ThrottlePolicyConstants.COLUMN_POLICY_ID));
                 globalPolicy.setUUID(rs.getString(ThrottlePolicyConstants.COLUMN_UUID));
-                globalPolicy.setTenantId(rs.getInt(ThrottlePolicyConstants.COLUMN_TENANT_ID));
+                globalPolicy.setTenantId(rs.getShort(ThrottlePolicyConstants.COLUMN_TENANT_ID));
```

Comment:
Is this relavent for this PR? @bhathiya has done this change to fix a issue. We should keep getInt

Reply:
I have changed it back to getInt.